src/Schema/AbstractSchema.php

```diff
@@ -574,4 +574,34 @@ public function getViewNames(string $schema = '', bool $refresh = false): array
 
         return (array) $this->viewNames[$schema];
     }
+
+    /**
+     * @throws Throwable
+     */
+    public function hasTable(string $tableName, string $schema = ''): bool
```

Comment:
`SchemaInterface::getTableNames()` has parameter `bool $refresh = false`, `SchemaInterface::hasTable()` also should have this parameter to pass it to `SchemaInterface::getTableNames()`

The same for other methods.

Reply:
Done

Reply:
Could you add tests with the `$refresh = true` argument?
And make all tests passed.

Reply:
Done.
Please keep in mind, that there are no test for this `$refresh` parameter for the parent functions `getTableNames()`, `getSchemaNames()` and `getViewNames()`.